builtin/staker_native.go

```diff
@@ -203,11 +212,16 @@ func init() {
 			env.ParseArgs(&args)
 			charger := gascharger.New(env)
 
-			err := Staker.NativeMetered(env.State(), charger).IncreaseStake(thor.Address(args.Endorsor), thor.Bytes32(args.ValidationID), args.Amount)
+			err := Staker.NativeMetered(env.State(), charger).
+				IncreaseStake(
+					thor.Address(args.Endorsor),
+					thor.Bytes32(args.ValidationID),
+					args.Amount,
+				)
 			if err != nil {
 				return []any{fmt.Sprintf("revert: %v", err)}
 			}
-			env.UseGas(thor.SstoreSetGas)
```

Comment:
Just to be sure it's not a mistake: is this gas consumption completely removed from this function call?

Reply:
Yes, I believe it was lingering on from previous code edits.

The thor.SstoreSetGas is spent inside the `IncreaseStake`